sherpa/optmethods/optfcts.py

```diff
@@ -85,20 +98,22 @@
 
 
 __all__ = ('difevo', 'difevo_lm', 'difevo_nm', 'grid_search', 'lmdif',
-           'minim', 'montecarlo', 'neldermead')
+           'minim', 'montecarlo', 'neldermead',
+           'InfinitePotential', 'EPSILON', 'FUNC_MAX',
```

Comment:
I would rather we don't export these by default yet. Not until we've gone through and resolved #2022.